Source/DafnyDriver/Commands/DafnyCliTests.cs

```diff
@@ -57,10 +60,13 @@ public static class DafnyCliTests {
     // We do not want absolute or relative paths in error messages, just the basename of the file
     "--use-basename-for-filename",
 
-    // Set a default time limit, to catch cases where verification time runs off the rails
-    "--verification-time-limit=300",
+    // Set a default resource limit, to catch cases where verification runs off the rails
+    "--resource-limit:50000000",
```

Comment:
```suggestion
    "--resource-limit:50e6",
```
?

(and likewise elsewhere)

Reply:
Yeah, I was debating doing that. It's probably better. :)